storage/object.go

```diff
@@ -41,21 +41,87 @@ func (yig *YigStorage) pickRandomCluster() (cluster backend.Cluster) {
 	return
 }
 
+func (yig *YigStorage) PickSpecificCluster(poolName string) (cluster backend.Cluster) {
```

Comment:
If only need to pick a specific cluster, why weight calculation is needed?

Reply:
maybe not only one ssd cluster